docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -9,24 +9,23 @@ import PaneHidden from '/img/guides/development-guides/How-to-show-and-hide-a-sp
 import PaneShown from '/img/guides/development-guides/How-to-show-and-hide-a-split-view-pane-with-MVVM/Pane-shown.PNG';
 
 # How To Show and Hide a Split View Pane with MVVM
-In this guide you will learn how to show and hide a split view panel using the MVVM pattern. If you are new to Avalonia UI please check out the following documents: [The MVVM patter](https://docs.avaloniaui.net/docs/concepts/the-mvvm-pattern/) - [Avalonia UI and MVVM](https://docs.avaloniaui.net/docs/concepts/the-mvvm-pattern/avalonia-ui-and-mvvm).
+In this guide you will learn how to show and hide a split view panel using the MVVM pattern. If you are new to Avalonia UI please check out the following documents: [The MVVM pattern](https://docs.avaloniaui.net/docs/concepts/the-mvvm-pattern/) - [Avalonia UI and MVVM](https://docs.avaloniaui.net/docs/concepts/the-mvvm-pattern/avalonia-ui-and-mvvm).
 To proceed in this guide you will need at least a basic understanding of what Data Binding is and how it works in Avalonia.
-If you are coming from WPF, then the mechanism is nearly the same, except for some differences that will not be covered in this guide.
 
 ## Starting
 First of all, you need to create an MVVM application project, which you can do by reading the following guide [Create and run a project](https://docs.avaloniaui.net/docs/get-started/test-drive/create-a-project).<br>
 Once you created the project you should be presented with the following files:
```

Comment:
I would probably rephrase this to: "The default MVVM project template will generate a project populated with the following default files:"